department-of-veterans-affairs/vets-website
# Burials | Labels for names must be descriptive

## Original Issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/42250

## URL(s) of changes
* Claimant information: [burials-and-memorials/application/530/claimant-information](/burials-and-memorials/application/530/claimant-information)
* Veteran information: [burials-and-memorials/application/530/veteran-information](/burials-and-memorials/application/530/veteran-information)
* Veteran previous names: [burials-and-memorials/application/530/military-history/previous-names](/burials-and-memorials/application/530/military-history/previous-names)

# Background
When applying for burial benefits, the veteran must be deceased, and therefore is not the one applying for benefits. The claimant is required to enter both their information and the deceased veteran's information. Part of the information required is the full name of both the claimant and deceased veteran. Currently, the labels for both sets of fields are labeled with `Your [x] name` where x is one of `first, middle, or last`. This doesn't accurately describe the input that is being asked for so the proposed change was:
* `Your [x] name` -> `Claimant's [x] name` for claimant information
* `Your [x] name` -> `Veteran's [x] name` for veteran information

There was also a desire to update the description text on the claimant information page.

## Steps to test
1. Enter `[<IP ADDRESS>](https://staging.va.gov/burials-and-memorials/application/530/claimant-information)` in browser
2. Start the form without signing in
3. View the claimant and deceased Veteran pages and confirm both ask for "your" name
4. On step 3, for the question "Did the Veteran serve under another name?" page, answer "yes" 

## Code changes
* Added a new component `ApplicantDescription` to house the new description text for the claimant information page
* Modified the fullNameUI object to use `Claimant's [x] name` for name labels on the `claimant-information` page
* Modified the fullNameUI object to use `Veteran's [x] name` for name labels on the `veteran-information` page

## How was your code tested
Tests still pass

## Acceptance criteria
- [ ] "Your [x] name" under claimant information is rewritten to "Claimant's [x] name" 
- [ ] First paragraph under step 1 of 6 is rewritten to: "As the person claiming burial benefits, you need to provide some key information. You aren’t required to fill in all fields, but we can review your application faster if you provide more information."
- [ ] "Your [x] name" under the deceased Veteran's information is rewritten to "Veteran's [x] name"
- [ ] "Your [x] name" under the deceased Veteran's previous names is rewritten to "Veteran's [x] name"

## Definition of done
